proto/kuksa/val/v1/val.proto

```diff
@@ -98,6 +98,7 @@ message SubscribeEntry {
 // Subscribe to changes in datapoints.
 message SubscribeRequest {
   repeated SubscribeEntry entries = 1;
+  optional uint64 interval_ms = 2;
```

Comment:
Findings:
* What is the meaning of "interval_ms"? It should be already clear from the parameter name. Currently, I cannot see (without reading the documentation) if it is minimum, typical, or maximum interval.
* Don't we add detailed documentation about the behaviour in the proto files? I agree with Erik (in the old PR): We should also describe the semantics in the interface definition. Imho, the "outside visible behaviour" always belongs to the contract description of an interface definition.
* In the current definition this parameter is related to the overall subscription, but not to certain signals - which is fine for me if the parameter defines the max update frequency. But the behaviour should be clearly defined if the the subscription comprises signals with different update frequencies.
* What is the behaviour, if signals are updated faster than the max frequency requested by the client: 
  * Does it get "just" some sample value at the moment the update interval is reached,
  * do we send some mean value of the last samples (how would that be calculated?),
  * do we accumulated values and send them as a batch,
  * ...?

If we are unsure about the behaviour to be achieved by this PR, we should rethink merging it ...

Reply:
Sorry, if this comment is not "fully" matching - I initially commented the corresponding PR in the old repo.
I also added a comment in the issue: https://github.com/eclipse/kuksa.val/issues/652